### PR TITLE
News post for playtest-20190209 and RAGL S7.

### PR DESCRIPTION
**Pull request by pchote**
TODO:
- [x] Battlefield News
- [x] Changelog link
- [x] Release IDs

### CODE REVIEW

File: content/news/playtest-20190209.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+A [new playtest build](/download/) is now available with a collection of fixes and improvements from [playtest-20190106](/news/playtest-20190106/).
```
Comment (obrakmann) [inline content/news/playtest-20190209.md]: If I remember correctly, some people have been confused with the "from playtest-xyz" syntax before.  Does "fixes to and improvements over playtest-20190106" work?

Comment (pchote) [reply content/news/playtest-20190209.md]: Done

**Comment by pchote** [comment]
Updated first sentence wording, Discord links, and added the release ID.

File: content/news/playtest-20190209.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+We are excited for these new features, which should go a long way towards improving combat fluidness and the ability to micromanage your forces.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190209-opportunityfire.gif" alt="Tanks targeting while moving">
+
+Tanks now automatically target enemy forces while moving.
+</div>
+
+The audio notification changes introduced in the last playtest have been refined and expanded by adding click feedback to the sidebar, reducing the volume of the "cash tick" sounds, and normalizing the volume changes between different music tracks. The build is rounded out with fixes for bugs and crashes that were reported in the first playtest &mdash; see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/7b4a50814a012c161e68378f7af533e6c2dfd5f7) and [playtest-20190106](/news/playtest-20190106/) news post for more detail.
```
Comment (matjaeck) [inline content/news/playtest-20190209.md]: Maybe better to put this into two sentences?

"The audio notification changes introduced in the last playtest have been refined and expanded by adding a sound feedback when clicking on items in the sidebar.  We also reduced the volume of the "cash tick" sounds, and normalized the volume changes between different music tracks."

Comment (matjaeck) [reply content/news/playtest-20190209.md]: nvm

**Pull request merged**